### PR TITLE
Credential Manifest types

### PR DESCRIPTION
**Pull request by amika-sq**
Adds Typescript types for Credential Manifests, and all of the various types which it contains.

### CODE REVIEW

File: react-native-ssi/src/types.ts [hunk 1]
```diff
@@ -96,3 +96,187 @@ export type TermsOfUse = {
...
+};
+
+export type CredentialIssuer = {
+  id: string;
+  name?: string;
+  styles?: EntityStyle;
+  comment?: string;
+};
+
+export type EntityStyleImage = {
```
Comment (shamilovtim) [inline react-native-ssi/src/types.ts:118]: I think the `Entity` gobbledygook can be removed

File: react-native-ssi/src/types.ts [hunk 2]
```diff
@@ -96,3 +96,187 @@ export type TermsOfUse = {
...
+  background?: EntityStyleColor;
+  text?: EntityStyleColor;
+};
+
+export type OutputDescriptor = {
+  id: string;
+  schema: string;
+  name?: string;
+  description?: string;
+  styles?: EntityStyle;
```
Comment (shamilovtim) [inline react-native-ssi/src/types.ts:139]: `styles?: OutputDescriptorStyle`

**Review by shamilovtim — APPROVED** [review]

File: react-native-ssi/src/types.ts (hunk 1 above)
Comment (amika-sq) [reply react-native-ssi/src/types.ts:118]: Maybe? It's implementing this spec: https://identity.foundation/wallet-rendering/v0.0.1/#entity-styles

Thinking we can just get rid of it altogether?

File: react-native-ssi/src/types.ts (hunk 2 above)
Comment (amika-sq) [reply react-native-ssi/src/types.ts:139]: Ahh, I see what you meant by the last comment now. The style is also defined on `CredentialIssuer`, which is why verite had it defined as `EntityStyle`.

File: react-native-ssi/src/types.ts (hunk 1 above)
Comment (shamilovtim) [reply react-native-ssi/src/types.ts:118]: Oh ignore what I said

**Pull request merged**